docs/1-Using-Fleet/3-REST-API.md

```diff
@@ -3068,3 +3033,94 @@ Modifies the osquery options configuration set in Fleet.
 ```
 
 ---
+
+## Fleet UI
+
+### List user's UI preferences
+
+Returns the selected user's UI preferences as a list of attributes.
+
+`GET /api/v1/kolide/users/app/attributes`
+
+#### Parameters
+
+None.
+
+#### Example
+
+`GET /api/v1/kolide/users/app/attributes?user_id=1`
+
+##### Default response
+
+`Status: 200`
+
+```
+{
+  "attributes": [
+    {
+      "user_id": 1,
+      "key": "hosts_table_columns"
+      "value": "[
+        { “label”: “hostname” },
+        { “label”: “status” },
+        { “label”: “osquery_version” },
+        { “label”: “ipv4” }
+      ]"
+    },
+  ]
+}
+```
+
+### Modify user's UI preferences
+
+Modifies the selected user's UI preferences for the supplied attribute.
+
+`POST /api/v1/kolide/users/app/attributes`
```

Comment:
for the route url, I think I would do something that riffs off of the "me" route